blocks/serialization.py

```diff
@@ -36,6 +41,7 @@
 
 def pickle_dump(*args, **kwargs):
     """A wrapper around pickle's dump that provides informative errors."""
+    kwargs.setdefault('protocol', DEFAULT_PROTOCOL)
```

Comment:
No, much faster! On python 2 it switches from protocol 0 to 2, on Python 3
nothing changes (keeps using protocol 3). Pickling time went from 5 minutes
to 10 seconds for me
On Mar 10, 2015 9:44 AM, "Dmitry Bogdanov" <notifications@github.com> wrote:

> In blocks/serialization.py
> <https://github.com/bartvm/blocks/pull/443#discussion_r26121974>:
>
> > @@ -36,6 +41,7 @@
> >
> >  def pickle_dump(*args, **kwargs):
> >      """A wrapper around pickle's dump that provides informative errors."""
> > +    kwargs.setdefault('protocol', DEFAULT_PROTOCOL)
>
> This is more memory efficient and slower, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bartvm/blocks/pull/443/files#r26121974>.
>